diff --git a/etc/ssh/sshd_config b/etc/ssh/sshd_config
index 0eba96d..af55ad3 100644
--- a/etc/ssh/sshd_config
+++ b/etc/ssh/sshd_config
@@ -33 +33 @@ Include /etc/ssh/sshd_config.d/*.conf
-#PermitRootLogin prohibit-password
+PermitRootLogin prohibit-password
@@ -38 +38 @@ Include /etc/ssh/sshd_config.d/*.conf
-#PubkeyAuthentication yes
+PubkeyAuthentication yes
@@ -42 +42 @@ Include /etc/ssh/sshd_config.d/*.conf
-AuthorizedKeysFile     .ssh/authorized_keys
+AuthorizedKeysFile     .ssh/authorized_keys /etc/sshd/authorized_keys.d/%u
@@ -58 +58 @@ AuthorizedKeysFile        .ssh/authorized_keys
-#PasswordAuthentication yes
+PasswordAuthentication no
